src/OpenTelemetry.AutoInstrumentation/Configurations/DelayedInitialization.cs

```diff
@@ -25,15 +25,21 @@ internal static class DelayedInitialization
 {
     internal static class Traces
     {
+#if NET462
```

Comment:
```suggestion
#if NETFRAMEWORK
```

for this I am pretty sure that it should be addressed :wink: